src/ralph/conf.py

```diff
@@ -126,6 +144,7 @@ class MongoDatabaseBackendSettings(InstantiableSettingsItem):
 class DatabaseBackendSettings(BaseModel):
     """Pydantic model for database backend configuration settings."""
 
+    ES_ASYNC: AsyncESDatabaseBackendSettings = AsyncESDatabaseBackendSettings()
```

Comment:
As suggested:

```suggestion
    ES_ASYNC: ESDatabaseBackendSettings = ESDatabaseBackendSettings()
```

Reply:
We still needs to overload the class path though.